src/components/buttons/accessible-button/README.md

```diff
@@ -39,19 +39,19 @@ export default Button;
-| isToggled        | Bool     |     -     | Set this prop to indicate a toggle button and it's state                                                      |
-| buttonRef        | Function |     -     | Used as button reference                                                                                      |
-| isToggleButton   | Bool     |     -     | Used to set `aria-pressed˚ state                                                                              |
-| isDisabled       | Bool     |     -     | Set this prop to indicate a disabled button                                                                   |
-| onClick          | Function |     ✓     | Callback that is called when the button is clicked                                                            |
-| className        | String   |     -     | Will be applied to the `<button/>` to allow for custom styling                                                |
-| buttonAttributes | Object   |     -     | Attributes to be applied to the `<button/>` element. Can be use to e.g. add data-\* attributes to the button. |
+| Property         | Type      | Required? | Description                                                                                                                                                                                  |
+| :--------------- | :-------- | :-------: | :------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| id               | String    |     -     | Used as the HTML `id` attribute.                                                                                                                                                             |
+| type             | String    |     -     | Used as the HTML `type` attribute. Supported values are `submit`, `reset` and `button`.                                                                                                      |
+| label            | String    |     ✓     | The text that should be used as `aria-label`                                                                                                                                                 |
+| children         | Node      |     ✓     | Button content                                                                                                                                                                               |
+| isToggled        | Bool      |     -     | Set this prop to indicate a toggle button and it's state                                                                                                                                     |
+| ref              | React ref |     -     | Used as button reference                                                                                                                                                                     |
```

Comment:
I changed that to use `React.createRef` and `React.forwardRef`. We actually never use this prop, so I'm not sure if we still need it or not...